leaderboard/service/get_top_percentage.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"github.com/topfreegames/podium/leaderboard/model"
+)
+
+const getTopPercentageServiceLabel = "get top percentage"
+
+// GetTopPercentage retrieves top x% members from the leaderboard.
+func (s *Service) GetTopPercentage(ctx context.Context, leaderboardID string, pageSize, amount, maxMembers int, order string) ([]*model.Member, error) {
+	if amount < 1 || amount > 100 {
+		return nil, fmt.Errorf("Percentage must be a valid integer between 1 and 100.")
```

Comment:
On other service errors that aren't handler by the functions we are using a customized Error `GeneralError`. I think that it is more appropriate here instead of generic `fmt.Errorf`. What do you think?